dydx/dydxPresenters/dydxPresenters/_v4/Profile/dydxProfileViewBuilder.swift

```diff
@@ -72,5 +73,10 @@ private class dydxProfileViewPresenter: HostedViewPresenter<dydxProfileViewModel
         balancesPresenter.$viewModel.assign(to: &viewModel.$balances)
 
         attachChildren(workers: childPresenters)
+
+        viewModel.share?.shareAction = {
+            Router.shared?.navigate(to: RoutingRequest(path: "/action/share", params: ["text": DataLocalizer.shared?.localize(path: "APP.GENERAL.SHARE_MESSAGE", params: nil) ?? "",
```

Comment:
Would be nice to add some tracking event for this action... maybe adding it to ShareAction's navigate() call?

Reply:
good suggestion, added e85447909b508c01a26d6a4715d872e6bb753d01
and tested:
![Screenshot 2024-02-27 at 5 14 18 PM](https://github.com/dydxprotocol/v4-native-ios/assets/149746839/f368ebf3-aad3-458a-b148-5aea21bfc992)

Reply:
have also added to the [notion tracking doc](https://www.notion.so/dydx/v4-Amplitude-Analytics-Events-d12c9dd791ee4c5d89e48588bb3ef702?pvs=4)